drivers/i2s/i2s_esp32.c

```diff
@@ -12,18 +12,28 @@
 #include <soc.h>
 #include <zephyr/drivers/clock_control.h>
 #include <zephyr/drivers/pinctrl.h>
+#include <zephyr/devicetree.h>
+
+#if !SOC_GDMA_SUPPORTED
```

Comment:
What is `SOC_GDMA_SUPPORTED` is set per SoC but `dma` is not enabled, i.e, app is using descriptors instead?